### PR TITLE
 Stylize gem push and notifier emails

### PR DESCRIPTION
**Pull request by sonalkr132**
Related: #1950
Before:
Screenshot from 2019-07-16 14-12-10

After:
Screenshot from 2019-07-16 14-11-48

### CODE REVIEW

File: app/views/mailer/gem_pushed.html.erb [hunk 1]
```diff
@@ -1,33 +1,56 @@
...
+<!-- Body -->
+<table width="100%" border="0" cellspacing="0" cellpadding="0" bgcolor="#ffffff">
+  <tr>
+    <td class="content-spacing" style="font-size:0pt; line-height:0pt; text-align:left" width="20"></td>
+    <td>
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="35" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <div class="h3-1-center" style="color:#1e1e1e; font-family:Georgia, serif; min-width:auto !important; font-size:20px; line-height:26px;">
+        <p>
+          A gem you have push access to, has recently released a new version.
```
Comment (simi) [inline app/views/mailer/gem_pushed.html.erb]: :+1:

**Comment by eliotsykes** [comment]
Ahem, well that "before" is embarassing...

I didn't recheck the email previews after rebasing and incorporating the layout improvements. 

Apologies and thanks for fixing and improving on 1950 overall, much appreciated @sonalkr132 😃 👍

File: app/views/mailer/gem_pushed.html.erb (hunk 1 above)
Comment (dwradcliffe) [inline app/views/mailer/gem_pushed.html.erb]:
```suggestion
          A gem you have push access to has recently released a new version.
```

Comment (sonalkr132) [reply app/views/mailer/gem_pushed.html.erb]: done

**Comment by sonalkr132** [comment]
Screenshot from 2019-07-16 20-18-57
we are getting empty `to` and no `bcc`. I am wondering if this could an issue for spam detectors. @eliotsykes Was there any strong argument for [using `bcc`](https://github.com/rubygems/rubygems.org/blob/master/app/mailers/mailer.rb#L50) instead of `to` or just iterating over owners?

**Comment by eliotsykes** [comment]
No strong argument, I only opted to send a single email to multiple recipients via BCC for 2 reasons:

1. Submit "one" email from the app to SendGrid. I assumed it'd be more efficient than sending one email to each owner.
2. Use BCC to minimize exposure of owner email addresses.

**Pull request merged**